burndown_chart.py.orig

```diff
@@ -0,0 +1,194 @@
+import argparse
+import datetime
+from redminelib import Redmine
+from redminelib.exceptions import ResourceAttrError
+from plotly.offline import plot
+import plotly.graph_objs as go
+
+dt_format = '%Y-%m-%d'
+team = 'y'
+dt_today = datetime.datetime.today()
+today = datetime.datetime(dt_today.year, dt_today.month, dt_today.day, 0, 0, 0, 0)
+
+<<<<<<< HEAD
```

Comment:
maybe you pushed some backup file accidentally?

Reply:
Ouch, let me fix that. Seems I've resolved some conflicts some time ago =))